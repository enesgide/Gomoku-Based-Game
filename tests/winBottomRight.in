place S19
place S18
place R18
place R17
place Q17
place Q16
place P16
place P15
place O15