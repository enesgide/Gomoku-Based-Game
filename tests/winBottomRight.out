Black wins!
S19S18R18R17Q17Q16P16P15O15
Thank you for playing!
